content/tbb/export-and-import-bookmarks/contents.lr

```diff
@@ -0,0 +1,53 @@
+In order to export, import, backup or restore bookmarks in Tor Browser, go to:
+
+- Hamburger menu >> Library >> Bookmarks >> Show All Bookmarks (below the menu)
+- From the toolbar on the Library window, click Import and Backup
+
+**If you wish to export bookmarks**
+- Choose Export Bookmarks to HTML
+- In the Export Bookmarks File window that opens, choose a location to save the file, which is named bookmarks.html by default. The desktop is usually a good spot, but any place that is easy to remember will work.
+- Click the Save button. The Export Bookmarks File window will close.
+- Close the Library window.
+
+> Your bookmarks are now successfully exported from Tor Browser. The bookmarks HTML file you saved is now ready to be imported into another web browser.
+
+**If you wish to Import bookmarks**
+- Choose Import Bookmarks from HTML
+- Within the Import Bookmarks File window that opens, navigate to the bookmarks HTML file you are importing and select the file.
+- Click the Open button. The Import Bookmarks File window will close.
+- Close the Library window.
+
+> The bookmarks in the selected HTML file will be added to your Tor Browser within the Bookmarks Menu directory.
+
+**If you wish to Backup**
+- Choose Backup
+- A new window opens and you have to choose the location to save the file. The file has a .json extension.
+
+**If you wish to Restore**
+- Choose Restore and then select the bookmark file you wish to restore. 
+- Click okay to the pop up box that appears and hurray, you just restored your backup bookmark.
+
+**Import Data from another browser**
+> Bookmarks can be transferred from Firefox to Tor Browser. There are two ways to export and import bookmarks in Firefox: [HTML file](https://support.mozilla.org/en-US/kb/export-firefox-bookmarks-to-backup-or-transfer) or JSON file. Remember any custom bridges and search engine additions.
+After exporting the data from the browser, follow the above step to import the bookmark file into your Tor Browser.
+
+We can also export Bookmarks from an old version of **Tor Browser for Android** and Import in a New version of the Tor Browser for Android following the steps below.
+
+1. Install regular esr or portable esr Firefox, of the same version of esr that Tor Browser used in the "old version" Tor Browser of your bookmarks.
+2. Drag you old places.sqlite (bookmarks file) into the Firefox profile (This will replace the new default places.sqlite).
+3. Start up the Firefox, open bookmarks manager, then export bookmarks as .html.
+4. Then start current Tor Browser, open bookmarks manager, then import that .html file. Those bookmarks will show up as a folder added at the end of the bookmarks manager tree pane.
+
+
+**BEWARE**  Because of the fact that the contents of the ../tor-browser-en-US/ directory will be cleared (the exact name depends on your particular language localisation), you should place the bookmarks .JSON backup file at a different location, otherwise you'll lose it.
```

Comment:
* [ ] You can remove **BEWARE** and include this as part of the instructions.
* [ ] Also that localization information is unnecessary. Rephrase in a way that you don't need to say that, for example "Because of the fact that the contents of the Tor Browser directory will be cleared, you should place the bookmarks file at a different location, otherwise you'll lose it."